applications/GeoMechanicsApplication/custom_processes/apply_boundary_phreatic_line_pressure_table_process.hpp

```diff
@@ -62,10 +62,9 @@ class ApplyBoundaryPhreaticLinePressureTableProcess : public ApplyConstantBounda
     /// this function will be executed at every time step BEFORE performing the solve phase
     void ExecuteInitializeSolutionStep() override
     {
-        KRATOS_TRY;
-        const int nNodes = static_cast<int>(mrModelPart.Nodes().size());
+        KRATOS_TRY
 
-        if (nNodes != 0) {
+        if (mrModelPart.Nodes().size() > 0) {
```

Comment:
Just for you to know, the `ModelPart` API has a method for this (`mrModelPart.NumberOfNodes`).

Reply:
Thanks @rubenzorrilla, I have already modified it and now I am using NumberOfNodes() instead of that. I am testing the code and will commit today.